README.md

```diff
@@ -1898,13 +1884,11 @@ Your program should avoid parsing the normal stdout since they may change in fut
 
 From a Python program, you can embed yt-dlp in a more powerful fashion, like this:
 
-```python
```

Comment:
This is not a good solution. We cannot just remove the language tag. `make_manpage` should perform the transformation instead

Reply:
Sorry, I meant prepare_manpage